docs/design/phase26-folded-interval-accumulator-spec.md

```
@@ -0,0 +1,59 @@
+# Phase 26: Folded Intervalized Decoding State-Relation Accumulator Spec
+
+## Goal
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)